src/amenities/amenities_server/Controllers/AmenitiesController.cs

```diff
@@ -0,0 +1,98 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace amenities_server.Controllers;
+
+[ApiController]
+[Route("amenities")]
+public class AmenitiesController : ControllerBase
+{
+    private IAmenityPersistence _amenityPersistence;
+
+    public AmenitiesController()
+    {
+        _amenityPersistence = Services.GetAmenityPersistence();
+    }
+
+    //get: /amenities
+    [HttpGet]
+    public IActionResult GetAmenities()
+    {
+        var amenities = _amenityPersistence.GetAmenities();
+
+        if (amenities == null)
+        {
+            return NotFound();
+        }
+
+        return Ok(amenities);
+    }
+
+    //get: /amenities/{id}
+    [HttpGet("{id}")]
+    public ActionResult<Amenity> GetAmenityByID(int id)
+    {
+        var amenity = _amenityPersistence.GetAmenityByID(id);
+
+        if (amenity == null)
+        {
+            return NotFound();
```

Comment:
Im not a fan of the default NotFound() response. It would be better to send our own message.
We don't have a defined error format, so this is going to define it for the entire project. Be thoughtful. A suggestion might be something like:

```
return new
{
    status = 404,
    message = "Resource not found",
    amenity-Id = id
};
```